aospy/data_loader.py

```diff
@@ -411,6 +452,7 @@ class GFDLDataLoader(DataLoader):
     """
     def __init__(self, template=None, data_direc=None, data_dur=None,
                  data_start_date=None, data_end_date=None,
+                 upcast_float32=None,
```

Comment:
Why `None` here but `True` in the other data loaders?  I think `True` would work here also.